Tests/UnitTests/Paywalls/Events/PaywallEventsRequestTests.swift

```diff
@@ -57,6 +57,33 @@ class PaywallEventsRequestTests: TestCase {
         assertSnapshot(matching: requestEvent, as: .formattedJson)
     }
 
+    func testCanInitFromDeserializedEvent() throws {
```

Comment:
This fails in main. It's very similar to the other test, but it uses the `EventsRequest.PaywallEvent(storedEvent:` initializer